images/gcloud-bazel/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Ubuntu with bazel, gcloud and its dependencies preinstalled.
+
+FROM ubuntu:16.04
```

Comment:
Could we use a google-maintained base image? 

https://console.cloud.google.com/launcher/details/google/ubuntu16_04

Reply:
Done